addons/l10n_latam_invoice_document/wizards/account_move_reversal.py

```diff
@@ -13,16 +13,21 @@ class AccountMoveReversal(models.TransientModel):
     l10n_latam_document_number = fields.Char(string='Document Number')
     l10n_latam_manual_document_number = fields.Boolean(compute='_compute_l10n_latam_manual_document_number', string='Manual Number')
 
-    @api.depends('l10n_latam_document_type_id')
+    @api.depends('l10n_latam_document_type_id', 'journal_id')
     def _compute_l10n_latam_manual_document_number(self):
         self.l10n_latam_manual_document_number = False
-        for rec in self.filtered('move_ids'):
-            move = rec.move_ids[0]
-            if move.journal_id and move.journal_id.l10n_latam_use_documents:
-                rec.l10n_latam_manual_document_number = move._is_manual_document_number()
+        for wiz in self.filtered(lambda x: x.journal_id and x.journal_id.l10n_latam_use_documents):
+            wiz.l10n_latam_manual_document_number = self.env['account.move'].new({
+                'move_type': wiz._reverse_type_map(wiz.move_ids.move_type),
```

Comment:
```suggestion
                'move_type': wiz._reverse_type_map(wiz.move_ids[0].move_type),
```
as the wizard may be opened with multiple moves, same for the other fields below :smile:

Reply:
Hi @antoine162 , thanks for the review!
In LATAM countries I think that if you try to open the wizard with multiple moves this pop up is raised: https://github.com/odoo/odoo/blob/755f930865b5c0fb41d7bd5cbf1f215ce2a06b79/addons/l10n_latam_invoice_document/wizards/account_move_reversal.py#L56
Do you think the change is still necessary? I'm not sure honestly.

Reply:
I agree! Thanks for the feedback. I already pushed the changes :)